mlf_core/create/template_creator.py

```diff
@@ -278,16 +282,17 @@ def create_common_files(self) -> None:
                                     'license': self.creator_ctx.license,
                                     'project_short_description': self.creator_ctx.project_short_description,
                                     'github_username': self.creator_ctx.github_username,
-                                    'creator_github_username': self.creator_ctx.creator_github_username},
+                                    'creator_github_username': self.creator_ctx.creator_github_username,
+                                    'framework': self.creator_ctx.language.capitalize() if domain == 'mlflow' else ''},
```

Comment:
I don't quite understand this. There are more domains than just mlflow (e.g. package).
As you maybe noticed I am using language and framework here interchangingly (which is wrong), but which is why you likely want to use framework here. This is my bad, but I don't think that there's a sane way to fix this.

Reply:
Yeah, the package cookiecutter.json does not have an attribute like `framework`, simply because its never needed there. I mean one could instead write `framework` as `language`? And, I mean, one could have also used a switch statement or similar here to make it more scalable as any new domains are added.

I only added this because of the README fyi, so I could refactor this out

Reply:
We could, but the package domain has a language again (python).
I was a little bit "lazy". Ideally it should have been

domain -> mlflow
language -> python
framework -> pytorch